aptos-move/aptos-vm/src/verifier/randomness.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::move_vm_ext::SessionExt;
+use aptos_types::transaction::EntryFunction;
+use move_binary_format::{
+    errors::{Location, VMResult},
+    CompiledModule,
+};
+
+/// Returns true if function has an annotation that it is unbiasable.
+pub(crate) fn is_entry_function_unbiasable(
+    session: &mut SessionExt,
+    entry_fn: &EntryFunction,
+) -> VMResult<bool> {
+    let module_bytes = session.load_module(entry_fn.module())?;
+    let module = CompiledModule::deserialize_with_config(
```

Comment:
I think we can avoid the deserialize? isn't there an API to load CompiledModule from move_vm? https://github.com/aptos-labs/aptos-core/blob/7e61ef1c605a4e799078b0200b863b25e27ae208/third_party/move/move-vm/runtime/src/move_vm.rs#L115

Reply:
Thanks! Done, we should probably clean up APIs to make sure that session can provide both bytes and compiled module